
Using AC_PROG_CXX conditionally fails with autoconf 2.57, see google.

--- php-4.3.1/configure.in.cxx	2003-02-15 21:41:38.000000000 +0000
+++ php-4.3.1/configure.in	2003-05-06 15:19:49.000000000 +0100
@@ -125,8 +125,8 @@
 
 AC_AIX
 
-dnl AC_PROG_CXX
-dnl AC_PROG_CXXCPP
+AC_PROG_CXX
+AC_PROG_CXXCPP
 
 dnl check for -R, etc. switch
 PHP_RUNPATH_SWITCH
--- php-4.3.1/acinclude.m4.cxx	2002-11-17 19:28:57.000000000 +0000
+++ php-4.3.1/acinclude.m4	2003-05-06 15:19:21.000000000 +0100
@@ -790,13 +790,7 @@
   fi
 ])
 
-AC_DEFUN([PHP_REQUIRE_CXX],[
-  if test -z "$php_cxx_done"; then
-    AC_PROG_CXX
-    AC_PROG_CXXCPP
-    php_cxx_done=yes
-  fi
-])
+AC_DEFUN([PHP_REQUIRE_CXX])
 
 dnl
 dnl PHP_BUILD_SHARED
